toast.yml

```diff
@@ -1,9 +1,10 @@
-image: gcr.io/windmill-public-containers/tilt-toast@sha256:a6717cfce82671ab4536a2a6d8bdbe83eb45dd5386e86f70fbec1d64b95dd7e6
+image: gcr.io/windmill-public-containers/tilt-toast@sha256:3596eb666cb026d1770ecef6712f988cacfa8085021877dea3a77bbafa17c79a
 tasks:
   proto:
     input_paths:
       - pkg/webview/view.proto
     output_paths:
       - pkg/webview/view.pb.go
       - pkg/webview/view.pb.gw.go
-    command: protoc -I. -I/usr/include -I"$(go env GOPATH)"/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis --grpc-gateway_out=logtostderr=true:. --go_out=plugins=grpc,paths=source_relative:. pkg/webview/view.proto && goimports -local github.com/windmilleng/tft -w pkg/webview/view.pb.go
+      - web/src/
+    command: mkdir -p web/src && protoc -I. -I/usr/include -I"$(go env GOPATH)"/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis --grpc-gateway_out=logtostderr=true:. --go_out=plugins=grpc,paths=source_relative:. --js_out=import_style=commonjs:web/src --grpc-web_out=import_style=typescript,mode=grpcwebtext:web/src pkg/webview/view.proto && goimports -local github.com/windmilleng/tft -w pkg/webview/view.pb.go && mv web/src/Pkg/webview/* web/src && mv web/src/pkg/webview/* web/src && rm -rf web/src/Pkg && rm -rf web/src/pkg
```

Comment:
can you move this into a shell script? this is becoming unreadable

Reply:
also, i don't totally understand why you have both import_style=commonjs and import_style=typescript. i would expect those to be mutually exclusive

Reply:
Yeah @nicks ... I was confused about that too. I was just following an example I found with GitHub search, but it seemed wrong to me.

Reply:
but anyways I moved it in to a "shell script" :)